stumpy/mstump.py

```diff
@@ -525,13 +569,32 @@ def multi_distance_profile(
         and the Euclidean distance, respectively. This parameter is ignored when
         `normalize == True`.
 
+    T_subseq_isconstant : numpy.ndarray or callable or list, default None
+        A parameter that is used to show whether a subsequence of a time series in `T`
+        is constant(True) or not. T_subseq_isconstant can be a 2D boolean numpy.ndarry
+        or it be a callable function, which will be applied to each time series in `T`.
+        It can also be list of length `T.shape[0]`, i.e. dimension size of `T`. In this
```

Comment:
Thanks for the suggestion! I was wondering if we should avoid `T.shape[0]`. Because, I think `T` can also be a dataframe in which each time series is provided as one column. So, in that case, the number of time series is `T.shape[1]`.